ingest-mongodb-public/src/sources.ts

```diff
@@ -1,38 +1,25 @@
-import { DevCenterProjectConfig } from "./DevCenterDataSource";
-import { SnootyProjectConfig } from "./snooty/SnootyDataSource";
-import { makeRstOnGitHubDataSource } from "./RstOnGitHubDataSource";
-import { DataSource } from "./DataSource";
-import { makeDevCenterDataSource } from "./DevCenterDataSource";
-import { prepareSnootySources } from "./snooty/SnootyProjectsInfo";
-import { makeGitDataSource } from "./GitDataSource";
 import {
+  DataSource,
+  makeDevCenterDataSource,
+  makeRstOnGitHubDataSource,
+  DevCenterProjectConfig,
+  makeGitDataSource,
   HandleHtmlPageFuncOptions,
   handleHtmlDocument,
-} from "./handleHtmlDocument";
-import { makeAcquitRequireMdOnGithubDataSource } from "./AcquitRequireMdOnGithubDataSource";
-import {
+  makeAcquitRequireMdOnGithubDataSource,
   MakeMdOnGithubDataSourceParams,
   makeMdOnGithubDataSource,
-} from "./MdOnGithubDataSource";
+} from "ingest/sources";
+import {
+  prepareSnootySources,
+  LocallySpecifiedSnootyProjectConfig,
+} from "ingest/sources/snooty";
 
 /**
   Async constructor for specific data sources -- parameters baked in.
  */
 export type SourceConstructor = () => Promise<DataSource | DataSource[]>;
```

Comment:
should this be in the "ingest" package?

Reply:
I don't think so. This is pretty implementation specific for snooty sources.

Reply:
maybe rename `SnootySourceConstructor` then?